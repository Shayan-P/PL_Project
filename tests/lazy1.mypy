def fib(a=10):
    if a==0:
        return 1;
    else:  
        print(8);
        return fib(a-1)+1;
        ;
    ;
def meaningless(a=10):
    return 0;
    ;
a=fib(6);
b=a+1;
print(meaningless(fib(5)*fib(7))*fib(8));
