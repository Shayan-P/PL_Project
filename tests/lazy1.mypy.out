(num-val 0)
'finished
